assets-api.md

```diff
@@ -14,113 +14,55 @@ Used to consolidate cost model data. Supported aggregation types are cluster and
 {% endswagger-parameter %}
 
 {% swagger-parameter in="path" name="accumulate" type="boolean" required="false" %}
-When set to 
-
-`false`
-
-, this endpoint returns daily time series data vs cumulative data. Default value is 
-
-`false`
-
-.
+When set to `false`, this endpoint returns daily time series data vs cumulative data. Default value is `false`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="path" name="disableAdjustments" type="boolean" required="false" %}
-When set to `true`, zeros out all adjustments from cloud provider reconciliation, which would otherwise change the totalCost. Default value is
-
-`false`.
+When set to `true`, zeros out all adjustments from cloud provider reconciliation, which would otherwise change the totalCost. Default value is`false`.
```

Comment:
```suggestion
When set to `true`, zeros out all adjustments from cloud provider reconciliation, which would otherwise change the totalCost. Default value is `false`.
```